setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = pysatMadrigal
+version = file: pysatMadrigal/version.txt
+url = https://github.com/pysat/pysatMadrigal
+author = Angeline G. Burrell, et al.
+author_email = angeline.burrell@nrl.navy.mil
```

Comment:
Should we be using individual emails here or the developer's email?

Reply:
I think the pysat development email would be better, but it currently isn't standard.  Happy to update it here.

Reply:
Opps, this is a new standard I made, isn't it? :P